src/components/season/season-table-header-participation.tsx

```diff
@@ -9,9 +9,14 @@ import { Tooltip } from "../ui/tooltip";
 function SeasonTableHeaderParticipation({
   seriesTracks,
 }: {
-  seriesTracks: { [key: number]: number };
+  seriesTracks: { [key: string]: number };
 }) {
   const { myTracks } = useIr();
+  Object.keys(seriesTracks).forEach((key) => {
```

Comment:
In React we can't mutate the values coming from the props, so in this case you need to create a copy object like:

```ts
const filteredTracks = Object.fromEntries(
  Object.entries(seriesTracks).filter(([key]) => !key.includes("_cars"))
);

const tracks = Object.values(filteredTracks).map(
  (trackId) => TRACKS_JSON[trackId.toString() as keyof typeof TRACKS_JSON]
);
```

Reply:
Noted! I'll try to apply this from now on. Thanks for the head up!